config/initializers/doorkeeper.rb

```diff
@@ -106,10 +106,33 @@
   # Under some circumstances you might want to have applications auto-approved,
   # so that the user skips the authorization step.
   # For example if dealing with a trusted application.
-  # skip_authorization do |resource_owner, client|
-  #   client.superapp? or resource_owner.admin?
-  # end
+  skip_authorization do |resource_owner, client|
+    client.application.name == ENV['AUTO_AUTHORIZE_APP']
```

Comment:
I suggest we have a flag application.autoauth == 1

Reply:
Then it will be better if I move it to a separated patch